modules/installation/pages/client-requirements.adoc

```diff
@@ -23,13 +23,15 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sls}-ES 6 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
-| {sls} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
+| {sls} for SAP 15 | {x86}_64, {ppc}                    | icon:check[role="green"]      | icon:check[role="green"]
```

Comment:
12 and 11 are supported as well. This was the reason why we removed the version.
I think PM need to come up with a rule about the granulartity of product names and versions.

Reply:
ok, I reverted that one.